gym/envs/safety/semisuper.py

```diff
@@ -15,13 +15,10 @@
 class SemisuperEnv(gym.Env):
     def step(self, action):
         assert self.action_space.contains(action)
-        self.monitor._before_step(action)
```

Comment:
This environment looks like it's overriding `step` rather than `_step` — that's a bug.